README.md

```diff
@@ -10,9 +10,9 @@
 ## Welcome to scalecube! if you are new to scalecube:
 [Please read ScaleCube-services Motivations and vision](https://github.com/scalecube/scalecube/wiki)
 
-## Overview
-ScaleCube, the art of scaling, in microservice architecture scalecube is a strategy in which components can scale on X, Y, Z axis. 
-ScaleCube project provides the tools to develop, test and scale microservice components in a distributed manner with ease.
+## Overview 
+ScaleCube project provides the tools to develop, test and scale distributed components in a cluster with ease.
+it provides a general propuse cluster-membership library based on [SWIM Membership protocol](http://www.cs.cornell.edu/~asdas/research/dsn02-swim.pdf) with gossip protocol as improvment. scalecube-cluster is built as such it is Embeddable and can be used by any cluster-aware application which requires cluster membership, node discovery, faliure-detection and gossip style of communication between member nodes. based on that unique properties scalecube implements aditional seperated module implementing microservices coordination library that features service discovery, fault-tollarance at scale.
```

Comment:
Please check typos (propuse = purpose, improvment = improvement etc.) and punctuation marks.